internal/service/networkcontainer/resource_network_container_test.go

```diff
@@ -136,7 +136,8 @@ func TestAccNetworkContainer_updateIndividualFields(t *testing.T) {
 		regionUpdated    = "EU_WEST_2"
 	)
 
-	resource.ParallelTest(t, resource.TestCase{
+	// Serial so it doesn't conflict with TestAccNetworkContainer_basicAWS
+	resource.Test(t, resource.TestCase{
```

Comment:
I understand this is not strictly related to this PR, but to fix potentially flaky from nightly run right?

Reply:
you're right, it's more related to the PR this is following-up, i'll update the description